src/hooks/useUploadsManager/useUploadsManager.tsx

```diff
@@ -170,10 +174,13 @@ export const UploadManagerProvider: React.FC = ({ children }) => {
               }
             },
           },
-          onUploadProgress: ({ loaded, total }: ProgressEvent) => {
-            updateAsset(asset.contentId, 'inProgress')
```

Comment:
Will this work without this? If we do a reconnect we will set status to `reconnecting`, I don't think there's anything that will change it back?

Reply:
If the `opts?.changeHost` is true it will set it again to `inProgress`, but yeah forgot about case if e.g. third try of reconnecting will succeed. Will add that